### ISSUE TITLE
Bokeh Logo with GridPlot

### ISSUE THREAD

**Issue by benoit9126** [bokeh/bokeh#3985]
Hi everybody,

I just remark that the Bokeh logo is missing when I display plots with a GridPlot object. Just look at one of the examples in the documentation:
```python
from bokeh.models import ColumnDataSource
from bokeh.plotting import figure, gridplot, output_file, show

output_file("brushing.html")

x = list(range(-20, 21))
y0 = [abs(xx) for xx in x]
y1 = [xx**2 for xx in x]

# create a column data source for the plots to share
source = ColumnDataSource(data=dict(x=x, y0=y0, y1=y1))

TOOLS = "box_select,lasso_select,help"

# create a new plot and add a renderer
left = figure(tools=TOOLS, width=300, height=300, title=None)
left.circle('x', 'y0', source=source)

# create another new plot and add a renderer
right = figure(tools=TOOLS, width=300, height=300, title=None)
right.circle('x', 'y1', source=source)

p = gridplot([[left, right]])

show(p)
```

The result can be seen at http://bokeh.pydata.org/en/latest/docs/user_guide/interaction.html
In fact, I do not know if it is a bug or a desired behaviour but it seems weird to have the logo everywhere else and not here.

**Comment by bryevdv** [comment]
Definitely a bug

**Comment by birdsarah** [comment]
Fixed as part of #4392. 

See an example here: https://s3.amazonaws.com/bokeh-travis/133456943/examples.html#plotting/file/grid
